src/main/java/org/folio/linked/data/service/SettingsService.java

```diff
@@ -0,0 +1,11 @@
+package org.folio.linked.data.service;
+
+import java.util.Optional;
+import org.folio.linked.data.domain.dto.SettingsItem;
+
+public interface SettingsService {
+
+  Optional<SettingsItem> getSetting(String scope, String key);
```

Comment:
Any need to expose this method in the interface? I believe only `isSettingEnabled` need to be exposed in the interface and `getSetting` can be a private method in the impl class.

Reply:
just wanted to make it public for future usages, but yes, we ain't gonna need it now

Reply:
changed to private